src/utils.ts

```diff
@@ -176,3 +176,33 @@ export function concatChildNodes(nodes: ZoweNode[]) {
 export function getAppName(isTheia: boolean) {
     return isTheia? "Theia" : "VS Code";
 }
+
+/*************************************************************************************************************
+ * Error Hanndling
+ * @param {errorDetails} error.mDetails
+ * @param {label} - additional information such as profile name, credentials, messageID etc
+ * @param {moreInfo} - additional/customized error messages
+ *************************************************************************************************************/
+export function errorHandling(errorDetails: any, label?: string, moreInfo?: string) {
```

Comment:
Is the intention to use the `utils.errorHandling` function in place of the regular `vscode.window.showErrorMessage`?
If so, I believe we missed a few occurrences.